pkgs/development/python-modules/ufl/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv
+, lib
+, fetchurl
+, python3Packages
+, dolfin
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "ufl";
+  inherit (dolfin) version;
+
+  src = fetchurl {
+    url = "https://bitbucket.org/fenics-project/ufl/downloads/ufl-${version}.tar.gz";
+    sha256 = "04daxwg4y9c51sdgvwgmlc82nn0fjw7i2vzs15ckdc7dlazmcfi1";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ numpy six ];
+
+  checkInputs = with python3Packages; [ pytest ];
+
+  checkPhase = ''
+    runHook preCheck
+    py.test test/
+    runHook postCheck
+  '';
+
+  meta = {
+    description = "A domain-specific language for finite element variational forms";
+    homepage = "https://fenicsproject.org/";
+    platforms = lib.platforms.all;
+    license = lib.licenses.lgpl3;
```

```suggestion
    license = licenses.lgpl3;
```

lgpl3Only or lgpl3Plus? 

Missing maintainers.

---

I have no idea how to maintain this. There wasn't a maintainer for this set of packages before either.

---

> There wasn't a maintainer for this set of packages before either.

Which is not great but alright.

---

`lgpl3Only` I believe. i may be wrong. Check [here](https://github.com/FEniCS/ufl/blo/3cdfb8acfeeb5404804e12a304b65f58c51d822c/COPYING#L75).